source/numerics.tex

```diff
@@ -12812,41 +12812,35 @@
 \begin{itemize}
 \item
 \tcode{remove_cvref_t<decltype(a.accessor().nested_accessor())>}
-if \tcode{Accessor} is a specialization of \tcode{conjugated_accessor}; otherwise,
+if \tcode{Accessor} is a specialization of \tcode{conjugated_accessor};
 \item
-\tcode{Accessor} if \tcode{remove_cvref_t<ElementType>} is an arithmetic type; otherwise,
+otherwise,
+\tcode{Accessor} if \tcode{remove_cvref_t<ElementType>} is an arithmetic type;
 \item
-\tcode{Accessor} if the expression \tcode{conj(E)} is not valid for any subexpression \tcode{E}
+otherwise,
+\tcode{conjugated_accessor<Accessor>}
+if the expression \tcode{conj(E)} is valid for any subexpression \tcode{E}
 whose type is \tcode{remove_cvref_t<ElementType>}
 with overload resolution performed in a context that includes the declaration
-\tcode{template<class T> conj(const T\&) = delete;}; and otherwise,
+\tcode{template<class U> conj(const U\&) = delete;};
```

Comment:
It's a shame we're losing the penultimate "and", though, isn't it?

Reply:
I wouldn't mind if you put the penultimate "and" back in : - ) .

Reply:
@jwakely Thoughts?

Reply:
Actually, thinking about it, I don't mind the lack of a penultimate "and" here.  This is an OR list (disjunction), not an AND list (conjunction).

Reply:
We don't put "and"s into "if... otherwise" ladders, which this is.